app.rb

```diff
@@ -1,9 +1,38 @@
 require 'sinatra/base'
+require_relative './lib/peep'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
```

Comment:
It's always a good idea to remove test code from code you submit for code review (anywhere, not just at Makers)